hat/assets/js/apps/Iaso/components/tables/TableFilters.js

```diff
@@ -61,18 +62,21 @@ const Filters = ({
         }
         onSearch(tempParams);
     };
+
+    const runsBaseUrl = 'orgunits/sources/links/runs';
```

Comment:
please use `baseUrls.algos` ! Also i would recommend  to use an array of urls were to use 3 columns.

Reply:
`TableFilters` is used by `SingleTable` a bit everywhere, it could ba a good solution also to drill down columns number through SingleTable and use 4 or 3 by default.